hedera-node/test-clients/src/main/java/com/hedera/services/bdd/junit/HapiTestEnv.java

```diff
@@ -33,8 +35,9 @@ public class HapiTestEnv {
     private static final int CAPTIVE_NODE_STARTUP_TIME_LIMIT = 300;
     private final List<HapiTestNode> nodes = new ArrayList<>();
     private final List<String> nodeHosts = new ArrayList<>();
+    private boolean started = false;
 
-    public HapiTestEnv(@NonNull final String testName, final boolean cluster) {
+    public HapiTestEnv(@NonNull final String testName, final boolean cluster, final boolean useInProcessAlice) {
```

Comment:
What will be the process/idea of using the 3rd argument(useInProcessAlice)? Currently, it's always **false** so we are always using SubProcessHapiTestNode. Is the idea to manually set it to **true** when we debug and use it only for that?

Reply:
Oh, I saw your comment in InProcessHapiTestNode